cmd/tkn/main.go

```diff
@@ -0,0 +1,21 @@
+// Copyright © 2019 The Knative Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import "github.com/tektoncd/cli/pkg/cmd"
+
+func main() {
+	cmd.Execute()
```

Comment:
nit: I would prefer having the only `os.Exit` in here instead of `pkg/cmd` :angel:

Reply:
did you mean something like `os.Exit(cmd.Execute() )`  ?